File: ttp-composite/windows/c/coverage/tcp_connect_9389/main.c
#include "../common.h"

int main(int argc, char **argv) {
    puts("COMPOSITE_CASE tcp_connect_9389");
    if (!fixture_begin(argc,argv,"tcp_connect_9389")) return 0;
    tcp_exchange(9389);
    success("tcp_connect_9389");
    return 0;
}

File: ttp-composite/windows/go/coverage/tcp_connect_9389/main.go
//go:build windows

package main

import (
	"fmt"
	"github.com/michaellong/telemetry-lab/ttp-composite/windows/go/coverage/internal/fixture"
	"github.com/michaellong/telemetry-lab/ttp-composite/windows/go/coverage/internal/netio"
)

func main() {
	fmt.Print("COMPOSITE_CASE tcp_connect_9389\n")
	if !fixture.Begin("tcp_connect_9389") {
		fixture.Hold()
		return
	}
	netio.Exchange("9389")
	fixture.Hold()
	fixture.Success("tcp_connect_9389")
}
